examples/sparse_encoder/training/distillation/README.md

```diff
@@ -1,2 +1,22 @@
 # Model Distillation
-TODO
\ No newline at end of file
+
+This page contains an example to make SparseEncoder models **faster, cheaper and lighter** through knowledge distillation. 
+
+Knowledge distillation allows us to compress knowledge from larger, more computationally expensive models (teacher models) into smaller, more efficient sparse models (student models). This approach can leverage bigger model results, including non-sparse models like Cross-Encoders and dense bi-encoders, to compress the knowledge into our small sparse model while maintaining much of the original performance.
+
+## MarginMSE
+**Training code: [train_splade_msmarco_margin.py](train_splade_msmarco_margin.py)**
+
+```{eval-rst}
+:class:`~sentence_transformers.sparse_encoder.losses.SparseMarginMSELoss` is based on the paper of `Hofstätter et al <https://arxiv.org/abs/2010.02666>`_. Like when training with :class:`~sentence_transformers.sparse_encoder.losses.SparseMultipleNegativesRankingLoss`, we can use triplets: ``(query, passage1, passage2)``. However, in contrast to :class:`~sentence_transformers.losses.MultipleNegativesRankingLoss`, `passage1` and `passage2` do not have to be strictly positive/negative, both can be relevant or not relevant for a given query.  
+
+The distillation process works by transferring knowledge from a powerful teacher model (like a Cross-Encoder ensemble) to our efficient sparse encoder student model. We compute the `Cross-Encoder <../../../cross_encoder/applications/README.html>`_ score for ``(query, passage1)`` and ``(query, passage2)`` using the teacher model. We provide scores for 160 million such pairs in our `msmarco-hard-negatives dataset <https://huggingface.co/datasets/sentence-transformers/msmarco-hard-negatives>`_, which contains pre-computed scores from a BERT ensemble Cross-Encoder. We then compute the distance: ``CE_distance = CEScore(query, passage1) - CEScore(query, passage2)``.
```

Comment:
Let's link to this one, it's simpler to use I think: https://huggingface.co/datasets/sentence-transformers/msmarco-scores-ms-marco-MiniLM-L6-v2
(it's the exact same data, just uploaded to a new dataset instead of being hidden as a pickle file)